### PR TITLE
Change "OS" to "operating system" per RHSSG

### PR DESCRIPTION
**Pull request by rh-max**
#### What changes are you introducing?
Substituting all occurences of "OS" for "operating system".

#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)
https://issues.redhat.com/browse/SAT-23961

#### Anything else to add? (Considerations, potential downsides, alternative solutions you have explored, etc.)
--

#### Checklists

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [x] Foreman 3.11/Katello 4.13
* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: guides/common/modules/proc_determining-hardware-and-os-configuration.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
-[id="Determining_Hardware_and_OS_Configuration_{context}"]
-= Determining hardware and OS configuration
+[id="Determining_Hardware_and_operating_system_Configuration_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_determining-hardware-and-os-configuration.adoc:1]: diff LGTM, please also rename the file to ensure that file name, anchor, and title matches.

File: guides/common/modules/proc_restricting-customrepoid-to-a-specific-os-version-or-architecture-in-project.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[id="Restricting_a_{customrepoid}_to_a_Specific_OS_Version_or_Architecture_in_Project_{context}"]
+[id="Restricting_a_{customrepoid}_to_a_Specific_operating_system_Version_or_Architecture_in_Project_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_restricting-customrepoid-to-a-specific-os-version-or-architecture-in-project.adoc:1]: same as above.

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
Hey Maxim, welcome again to Foreman docs. Diff LGTM minus two file names. In foreman-documentation, we always ensure that the anchor, title, and file name match (minus "proc_" etc prefixes)

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by Lennonka** [comment]
This is a style modification, therefore, no tech review should be needed and I'm setting the `tech review done` label.

**Comment by Lennonka** [comment]
@rh-max Unfortunately, I cannot even cherry pick this to 3.12. There are conflicts. Please, file a new PR on the stable branch if you think it's important to get it to older branches as well.

**Comment by rh-max** [comment]
> @rh-max Unfortunately, I cannot even cherry pick this to 3.12. There are conflicts. Please, file a new PR on the stable branch if you think it's important to get it to older branches as well.

Thanks, created the cherrypick PR: https://github.com/theforeman/foreman-documentation/pull/3440

**Pull request merged**